src/main/scala/nl/knaw/dans/lib/logging/servlet/ServletLogger.scala

```diff
@@ -50,6 +50,10 @@ trait AbstractServletLogger {
    *   class ExampleServlet extends ScalatraServlet with ServletLogger with DebugEnhancedLogging {
    *     get("/") {
    *       logResponse(Ok("All is well"))
+   *       // TODO review comment:
+   *       // When reading this line the main action (create a response)
+   *       // drowns in the side effect (log the created response).
+   *       // Do you have a good use case for it?
```

Comment:
I don't have an immediate use case for it. I'm just saying that this is a possibility. However, I do agree with the drowning, you mentioned. Would it help if we changed it to:

```scala
class ExampleServlet extends ScalatraServlet with ServletLogger with DebugEnhancedLogging {
  get("/") {
    logResponse {
      Ok("All is well")
    }
  }
}
```

Or would you say it is even better to have this whole example removed and only list the `Ok("All is well").logResponse` example?